### PR TITLE
Mp/merge bifrost 3

### PR DESCRIPTION
**Pull request by marcparadise**
Not for merge yet, but I did want to get this through travis and see how it behaves.

### CODE REVIEW

File: omnibus/config/projects/chef-server.rb [hunk 1]
```diff
@@ -70,10 +70,6 @@
 dependency "keepalived"
 dependency "bookshelf"
 
-# the front-end services
-dependency "oc_bifrost"
-dependency "oc_id"
```
Comment (stevendanna) [inline omnibus/config/projects/chef-server.rb]: Did you mean to remove oc_id?

Comment (marcparadise) [inline omnibus/config/projects/chef-server.rb]: d'oh, nice catch

File: omnibus/omnibus.rb [hunk 2]
```diff
@@ -11,12 +11,12 @@
...
+build_retries 0 #TODO NOPE
 fetcher_read_timeout 120
 
 # Load additional software
 # ------------------------------
 # software_gems ['omnibus-software', 'my-company-software']
 # local_software_dirs ['/path/to/local/software']
 
-fatal_transitive_dependency_licensing_warnings true
+fatal_transitive_dependency_licensing_warnings  false
```
Comment (stevendanna) [inline omnibus/omnibus.rb]: We need to leave this to true in the long run.  What is failing currently?

**Comment by stevendanna** [comment]
Overall I think this is probably a good idea.  One less unix-level service from the user's perspective.  Do you have a sense of what the TODO list here is @marcparadise?

**Comment by marcparadise** [comment]
Just some likely squashing and more eyes. Upgrade testing to make sure services are properly cleaned up.

There's also a sqerl branh that's part of this that needs to land, I'm going to get that in this week.

**Comment by stevendanna** [comment]
I still think this could make a lot of sense, but I'm going to close out the PR (will leave branch though) as this is now over a year old.

**Pull request closed without merging**